labs/blockchain-explorer-iroha.md

```diff
@@ -0,0 +1,17 @@
+# Lab Name
+[Integration of Hyperledger Iroha into Hyperledger Explorer tool](https://wiki.hyperledger.org/display/INTERN/Integration+of+Hyperledger+Iroha+into+Hyperledger+Explorer+tool)
```

Comment:
Lab name should be what we will end up calling the github repository. Maybe something more like iroha-explorer would be appropriate.